README_USER_GUIDE.md

```diff
@@ -120,7 +120,7 @@ Example of a successful response:
 ## Caveats
 
 - If there is an ongoing node upgrade, the node upgrade fails with 409. However, this should not abort the playbook.
-- Upgrading the same package on a node that has previously finished upgrading does not fail on upgrade.
+- Attempting to upgrade compute nodes already running the firmware versions included in the package (i.e compute firmware bundle) will result in the compute nodes going through the upgrade stages and subsequently report a successful upgrades for the playbook. The compute nodes will be placed into maintenance mode and rebooted to apply compute firmware bundle, even though the actual component-level firmware upgrades stage will be skipped.
```

Comment:
Could you please remove this statement: "even though the actual component-level firmware upgrades stage will be skipped.", since the component level firmware upgrade is something that we don't want to mention. The team confirms that the component-level firmware upgrade does happen even when the version are the same also there is no checks in the services level python code for the same. @tim-ntap